examples/jsm/loaders/obj2/shared/MeshReceiver.js

```diff
@@ -9,7 +9,7 @@ import {
 	LineSegments,
 	Mesh,
 	Points
-} from "../../../../build/three.module.js";
+} from "../../../../../build/three.module.js";
 
 import { MaterialHandler } from "./MaterialHandler.js";
```

Comment:
This seems to be an unused import.

Reply:
The constructor is never invoked here, but `getMaterial` is used. If `MeshReceiver` is used without `OBJLoader2` linking them this will be a problem right (code is missing then). Other language compilers will not allow this. JS is fine, but what is a good practice here?

Reply:
I will remove it for now, but I welcome a suggestion how to handle such a situation.